tests/v2/validation/charts/cisbenchmark_test.go

```diff
@@ -0,0 +1,298 @@
+package charts
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/rancher/shepherd/clients/rancher"
+	"github.com/rancher/shepherd/clients/rancher/catalog"
+	management "github.com/rancher/shepherd/clients/rancher/generated/management/v3"
+	v1 "github.com/rancher/shepherd/clients/rancher/v1"
+	"github.com/rancher/shepherd/extensions/charts"
+	"github.com/rancher/shepherd/extensions/clusters"
+	"github.com/rancher/shepherd/pkg/config"
+	namegen "github.com/rancher/shepherd/pkg/namegenerator"
+	"github.com/rancher/shepherd/pkg/session"
+	"github.com/sirupsen/logrus"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+)
+
+const (
+	scanError        = "error"
+	scanPending      = "pending"
+	scanRunning      = "running"
+	scanReporting    = "reporting"
+	CisConfigFileKey = "cis"
+	ClusterScanYaml  = `
+apiVersion: cis.cattle.io/v1
+kind: ClusterScan
+metadata:
+  name: %s #string
+  annotations:
+    {}
+#    key: string
+spec:
+  scanProfileName: %s
+  scheduledScanConfig:
+    retentionCount: 3
+    scanAlertRule:
+#      alertOnComplete: boolean
+#      alertOnFailure: boolean
+#    cronSchedule: string
+  scoreWarning: pass
+  `
+)
```

Comment:
usually consts like this are put in a file.yml instead of as a string variable

Reply:
Done!